code/modules/jobs/job_types/paramedic.dm

```diff
@@ -26,9 +26,9 @@
 	ears = /obj/item/radio/headset/headset_med
 	gloves = /obj/item/clothing/gloves/color/latex/nitrile
 	uniform = /obj/item/clothing/under/rank/medical/paramedic
-	mask = /obj/item/clothing/mask/cigarette
 	shoes = /obj/item/clothing/shoes/jackboots
 	head = /obj/item/clothing/head/soft/emt
+	mask = null //Skyrat Edit
```

Comment:
Comment this out broje, making it null is a bit hacky and is gonna fuck with a lot of things.

Reply:
Useroth himself told me to do it like that lmao

Reply:
I confirm. It wouldn't fuck with things, but I agree that would be hacky.